### ISSUE TITLE
Screen Reader only styles are not recognized by AXE DevTools (and throw errors)

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#1556]
Description  
---
AXE DevTools is the main automatic accessibility checker. Their implementation of checking if an element is visually hidden but accessible for screen readers consists by checking 3 CSS styles:
```
.sr-only {
  position: absolute;
  overflow: hidden;
  height: 1px;
}
```

Some of our components, e.g. `Link` and `Link-Social` are missing the `position: absolute` style which has the effect that AXE doesn't recognize these elements as SR-Only. This leads to false positives and AXE fires some issues regarding contrasts. Thus we need to apply the `@text-hidden` mixin.

- https://github.com/dequelabs/axe-core/issues/2962 
- https://github.com/dequelabs/axe-core/pull/2985/files

Expected behavior
---
- [-] Apply mixin to Link/Button (Hidden Label) and Link Social
--> Not possible to resolve because these components need another type of visually hidden text due to component architecture. And this implementation doesn't get recognized by AXE as sr-only state and will fire a possible contrast issue which has to be checked manually.

Screenshots
---
<img width="1132" alt="Bildschirmfoto 2021-08-23 um 14 36 00" src="https://user-images.githubusercontent.com/40788023/130448204-686eff11-1f6d-47a5-946d-61e0156aa783.png">

Test Environment
---
Platform/Browser/Version: [e.g. iOS Safari 11, macOS Chrome 81]
